Platform.PCL/Realm.Sync.PCL/RealmExtensionsPCL.cs

```diff
@@ -18,8 +18,16 @@
 
 namespace Realms.Sync
 {
+    /// <summary>
+    /// A set of extension methods that provide Sync-related functionality on top of Realm classes.
```

Comment:
We should annotate extension classes with `[Browsable(false)]` so that they don't show up in code completion instead of documenting them.